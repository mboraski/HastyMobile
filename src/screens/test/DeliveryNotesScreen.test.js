// import 'react-native';
// import React from 'react';
// import { shallow } from 'enzyme';
// import configureStore from 'redux-mock-store';
//
// import DeliveryNotesScreen from '../DeliveryNotesScreen';
//
// const initialState = {
//     checkout: { notes: 'notes' }
// };
//
// describe('DeliveryNotesScreen', () => {
//     const middlewares = [];
//     const mockStore = configureStore(middlewares);
//     it('renders correctly', () => {
//         const wrapper = shallow(<DeliveryNotesScreen />, {
//             context: { store: mockStore(initialState) }
//         });
//         const render = wrapper.dive();
//         expect(render).toMatchSnapshot();
//     });
// });
