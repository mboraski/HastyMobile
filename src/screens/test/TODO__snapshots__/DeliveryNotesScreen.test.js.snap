// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeliveryNotesScreen renders correctly 1`] = `
<View
  keyboardVerticalOffset={16}
  onLayout={[Function]}
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
      },
      Object {
        "paddingBottom": 0,
      },
    ]
  }
>
  <Connect(ReduxForm) />
</View>
`;
